src/cardMarkup.ts

```diff
@@ -111,11 +111,14 @@ export function formatBodyForCard(
 
 	if (endOfEventMarkerIndex > 0) body = body.slice(0, endOfEventMarkerIndex);
 	// Remove external image links
+        body = body.replace(/!\[.*\]\(.*\)/gi, "");
+        // Remove tags
+        if ( settings.showNoteHeadersInCardBody )
+            body = body.replace(/#[a-zA-Z\d-_/]*/gi, "");
+        else
+            body = body.replace(/#.*/gi, ""); // remove markup headers
```

Comment:
Could you explain the reasoning behing the condition check here ?

Reply:
The condition check is for the header flag from settings
if set it uses the original regexp to only remove hashes from the text, leaving the rest of it
    ("## Background" --> "Background")
if not set the regexp removes the entire header line line from hash onwards.
  ("## Background " --> "" )

This is where the text for the card is generated and was the easiest place I could see for the chaneg to be made.